src/vpic/vpic.h

```diff
@@ -500,7 +500,7 @@ class vpic_simulation {
         max_local_nm = 16*(MAX_PIPELINE+1);
     }
     return append_species( species( name, (float)q, (float)m,
-                                    (int)max_local_np, (int)max_local_nm,
+                                    (size_t)max_local_np, (size_t)max_local_nm,
```

Comment:
Did we ever get to the bottom of which type is right here?

Reply:
I guess not.  If you grep through the code, is there ever any indication that either max_locap_np
or max_local_nm needs to be a signed int?  If not, I would opt for using size_t.  What do you
think?

Reply:
That's fine, and I don't disagree, but do we need to promote it to 64bit (as using size_t does)?